gateway/src/event.rs

```diff
@@ -105,6 +105,19 @@ bitflags! {
         const SHARD_RECONNECTING = 1 << 37;
         /// Shard is resuming a session with the gateway.
         const SHARD_RESUMING = 1 << 38;
+        /// Sent when a thread is created, relevant to the current user,
```

Comment:
The "sent when" is a bit redundant here.

Reply:
Copied these comments directly from Discord's docs, I'll rephrase them